app/views/developers/_form.html.erb

```diff
@@ -11,6 +11,17 @@
     </div>
   <% end %>
 
+  <div class="field">
+    <%= form.label :email %><br />
+    <%= form.email_field :email, { autofocus: true } %>
+
+    <% unless @developer.errors.full_messages.empty? %>
```

Comment:
`unless` + `empty?` should be `if` and `present?`
Try to use positive sentences. It's more readable.